zimply/zim_core.py

```diff
@@ -1000,6 +998,28 @@ def suggest(self, query, start=0, end=9, separator=" "):
     def get_suggestions_results_count(self, query, separator=" "):
         return self.get_search_results_count(query, separator)
 
+    def _tokenize_search_query(self, query, separator=" "):
```

Comment:
What about adding a docstring to the method to easily explain what it outputs? It could be in a [doctest](https://docs.python.org/3/library/doctest.html) valid format:
```suggestion
    def _tokenize_search_query(self, query, separator=" "):
    """
    >>> self._tokenize_search_query(None, "potato chips")
    (['potato', 'chips'], '"potato" * "chips" *')
    
    >>> self._tokenize_search_query('"potato chips" crunchy')
    (['potato chips', 'crunchy'], '"potato chips" * "crunchy" *')
    """
```

Reply:
Looking good. As far as I'm concerned this is ready to be accepted. 

@manuq suggested some doctest but I'm not sure I am in favour of that; there is no doctest used in ZIMply, so it feels weird to start that for this update. If you do want to add another line of generic comments let me know. Otherwise I will accept this pull request.

Reply:
Yeah, I'll skip the doctest part since it sticks out oddly (although, ooooh, I like that thing). But I added a docstring with some more detailed information.

Reply:
Sounds good!